x-pack/libbeat/licenser/license.go

```diff
@@ -27,7 +25,7 @@ import (
 // mode is the license in operation. (effective license)
 // status is the type installed is active or not.
 type License struct {
-	UUID        uuid.UUID   `json:"uid"`
+	UUID        string      `json:"uid"`
```

Comment:
Should this be named `UID` rather than UUID? Perhaps the name is placing more meaning on the value than what's "guaranteed" by the license interface (loosely speaking since I don't know that there is any spec for this license).

Reply:
I will keep it UUID for now